examples/collection-only/collection.json

```diff
@@ -1,7 +1,12 @@
 {
   "type": "Collection",
   "stac_version": "1.0.0-rc.2",
-  "stac_extensions": [],
+  "stac_extensions": [
```

Comment:
We'd need to make the following lines a separate PR if we want to merge this now. CI fails due to this change and the following issue: https://github.com/stac-extensions/scientific/pull/2

Still, these changes should be in RC3...

Reply:
Ok, I think I didn't quite understand what you were saying on the call @m-mohr, but I think I've got what you wanted, though feel free to correct. I made a PR https://github.com/radiantearth/stac-spec/pull/1103 to do the scientific change, that we'll keep as an open PR until we have scientific 1.0.1. 

And then just need to remove the scientific stuff from here. I would just edit directly, but guess I'll do as a PR since I'm not 100% this is the path you wanted.

Reply:
And PR to do the removal is #1104, just against this branch.